lib/src/deprecation.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Google LLC. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:collection/collection.dart';
+import 'package:pub_semver/pub_semver.dart';
+
+/// Represents a deprecated feature in the language.
```

Comment:
Nit: Remove "Represents", since an enum is always a representation of something

Reply:
Done